include/mbedtls/cipher.h

```diff
@@ -530,11 +544,12 @@ static inline mbedtls_operation_t mbedtls_cipher_get_operation( const mbedtls_ci
 /**
  * \brief               This function sets the key to use with the given context.
  *
- * \param ctx           The generic cipher context. May not be NULL. Must have
- *                      been initialized using mbedtls_cipher_info_from_type()
- *                      or mbedtls_cipher_info_from_string().
- * \param key           The key to use.
- * \param key_bitlen    The key length to use, in bits.
+ * \param ctx           The generic cipher context. This must be initialized
+ *                      using mbedtls_cipher_info_from_type() or
```

Comment:
This comment doesn't fit, because `mbedtls_cipher_info_from_type()` returns a cipher information structure, but cannot be used to initialized a cipher context.

Reply:
Suggestion: This must be initialized and bound to a cipher information structure.